core/lib/multivm/src/versions/vm_latest/constants.rs

```diff
@@ -124,7 +136,16 @@ pub(crate) const TX_OPERATOR_L2_BLOCK_INFO_OFFSET: usize =
 
 pub(crate) const TX_OPERATOR_SLOTS_PER_L2_BLOCK_INFO: usize = 4;
 pub(crate) const TX_OPERATOR_L2_BLOCK_INFO_SLOTS: usize =
-    (MAX_TXS_IN_BLOCK + 1) * TX_OPERATOR_SLOTS_PER_L2_BLOCK_INFO;
+    (MAX_TXS_IN_BATCH + 1) * TX_OPERATOR_SLOTS_PER_L2_BLOCK_INFO;
 
 pub(crate) const COMPRESSED_BYTECODES_OFFSET: usize =
     TX_OPERATOR_L2_BLOCK_INFO_OFFSET + TX_OPERATOR_L2_BLOCK_INFO_SLOTS;
+
+/// The maximal gas limit that gets passed into an L1->L2 transaction
```

Comment:
maybe mention what are the constraints here (why is it 72M and not higher or lower)